examples/guestbook/README.md

```diff
@@ -63,18 +68,58 @@ The web front end interacts with the redis master via javascript redis API calls
 
 **Note**:  If you are running this example on a [Google Container Engine](https://cloud.google.com/container-engine/) installation, see [this Container Engine guestbook walkthrough](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead. The basic concepts are the same, but the walkthrough is tailored to a Container Engine setup.
 
-### Step Zero: Prerequisites
+### Prerequisites
+
+This example requires a running Kubernetes cluster. See the [Getting Started guides](../../docs/getting-started-guides/) for how to get started. And follow the [Prerequisites](../../docs/user-guide/prereqs.md) to make sure your `kubectl` is ok. As noted above, if you have a Google Container Engine cluster set up, go [here](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
+
+### Quick Start
+
+This section shows a simplest way to get the example work. If you want to know the details, you should pass this and read the rest of the example.
```

Comment:
Change "pass this and read the rest of the example" to `skip this and read [the rest of the example](#step-one-start-up-the-redis-master)`